_docs/tasks/basic-access-control.md

```diff
@@ -45,31 +45,56 @@ of the `reviews` service. We would like to cut off access to version `v3` of the
    If you log in as any other user (or logout) you should see red ratings stars with each review,
    indicating that the `ratings` service is being called by the "v3" version of the `reviews` service.
 
-1. Explicitly deny access to version `v3` of the `reviews` service. 
+1. Explicitly deny access to version `v3` of the `reviews` service.
 
-   ```bash
-   istioctl mixer rule create global ratings.default.svc.cluster.local -f samples/apps/bookinfo/mixer-rule-ratings-denial.yaml
+   Before setting up the deny rule, we must create a handler and an instance definition that can be used in the deny rule.
+   Replace `metadata.namespace` with the appropriate value.
+   ```yaml
```

Comment:
We used to be able to import artifacts from the repo directly. @geeknoid is out, do you remember any examples?

Reply:
I have changed it to use direct links, here is how you do it.

```
  <pre data-src="https://raw.githubusercontent.com/istio/istio/release-0.1/samples/apps/bookinfo/rules/mixer-rule-ratings-ratelimit.yaml"></pre>
```